R/init.R

```diff
@@ -16,15 +16,15 @@
 #'
 #' @param data (`teal_data`, `teal_data_module`, `named list`)\cr
 #' `teal_data` object as returned by [teal.data::teal_data()] or
-#' `teal_data_modules` or simply a list of a named list of objects
+#' `teal_data_module` or simply a list of a named list of objects
 #' (`data.frame` or `MultiAssayExperiment`).
 #' @param modules (`list`, `teal_modules` or `teal_module`)\cr
 #'   nested list of `teal_modules` or `teal_module` objects or a single
 #'   `teal_modules` or `teal_module` object. These are the specific output modules which
 #'   will be displayed in the teal application. See [modules()] and [module()] for
 #'   more details.
-#' @param title (`NULL` or `character`)\cr
-#'   The browser window title (defaults to the host URL of the page).
+#' @param title (`shiny.tag` or `character`)\cr
```

Comment:
I'm not sure allowing _any_ `shiny.tag` is a good idea. Consider having `build_app_title` add a special class to check. (Emphasis on consider.)

Reply:
That's a good catch. Have to restrict to a sensible tag and document it as well. I'll add a change for this.

Reply:
Added a validation when `shiny.tag` is the input and improved the docs. PTAL now.